src/code/client.ts

```diff
@@ -1049,7 +1069,7 @@ class CloudFileManagerClient {
       interval = Math.round(interval / 1000)
     }
     if (interval > 0) {
-      return this._autoSaveInterval = window.setInterval((() => { if (this.shouldAutoSave()) { return this.save() } }), interval * 1000)
+      return this._autoSaveInterval = window.setInterval((() => { if (this.shouldAutoSave()) { console.log('autosaving'); return this.save() } }), interval * 1000)
```

Comment:
This new `console.log` is going to add a lot of noise to the console as it will print each second.

Reply:
I presume this can be removed.